@re-/model/docs/snippets/declaration/declaration.ts

```diff
@@ -1,10 +1,29 @@
+// @ts-nocheck
```

Comment:
So the problem here is I haven't updated the corresponding @re-/model test to actually import this stuff yet, but when I do it will break.

Unfortunately, we have to keep the `.js` suffixes at least in our snippets. When I mentioned we could just default to no suffixes for our demos, I meant we could add an extract snippet transformer for all TS import statements ending in `.js` to remove the `.js` suffix.

Once that's done, we can remove all the `@ts-nocheck` directives as well, which I didn't even know worked on TS files and is the scariest shit I've ever seen 🤣 I do totally understand why you did it though to avoid polluting the snippets with `@ts-ignore`.